app/helpers/application_helper.rb

```diff
@@ -79,6 +79,7 @@ def locale_name_pairs
   end
 
   def insert_extras(body)
+    body = NodeShared.notes_inline_grid(body)
```

Comment:
Wow, this looks great... but I think there should be a lot of overlap with the `notes_grid` method... do you think we could consolidate those two? In theory it should be relatively close ActiveRecord queries, but displayed differently, don't you think?

Reply:
Yes, makes sense!! I see the methods `notes_grid` and `notes_inline_grid` differ only in rendering the templates. Both render different templates having same active-record queries. So, let's keep a single function with a parameter (say thumbnails=false by default) and change the template accordingly?